<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetStudy</class>
 <widget class="QWidget" name="WidgetStudy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="stationLabel">
        <property name="text">
         <string>Station</string>
        </property>
        <property name="buddy">
         <cstring>stationEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="stationEdit">
        <property name="text">
         <string>Example</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="opModeLabel">
        <property name="text">
         <string>Op mode</string>
        </property>
        <property name="buddy">
         <cstring>opModeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="opModeEdit">
        <property name="text">
         <string>X</string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="blinkChecker">
        <property name="text">
         <string>Blinking</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="mrw::ui::BarWidget" name="barWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="mrw::ui::CircleLiveWidget" name="liveWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="mrw::ui::ClockWidget" name="clockWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="mrw::ui::OpModeWidget" name="opModeWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="mrw::ui::StationWidget" name="stationWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mrw::ui::BarWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/barwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mrw::ui::StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/stationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mrw::ui::ClockWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/clockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mrw::ui::CircleLiveWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/circlelivewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mrw::ui::OpModeWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/opmodewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
